File: 백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
      Scanner s = new Scanner(System.in);
      int N = s.nextInt();
      int M = s.nextInt();
      
    if(M <= 2){
        System.out.println("NEWBIE!");
    }else if(2 < M && M <= N){
        System.out.println("OLDBIE!");
    }else{
        System.out.println("TLE!");
    }
    }
}

File: 백준/Bronze/19944. 뉴비의 기준은 뭘까？/뉴비의 기준은 뭘까？.cc
#include <stdio.h>

int main(){
    int N, M;
    scanf("%d %d", &N, &M);
    if(M <= 2){
        printf("NEWBIE!");
    }else if(2 < M && M <= N){
        printf("OLDBIE!");
    }else{
        printf("TLE!");
    }
}